Recast/iTunesPodcasts/Models/RSS/Cloud.swift

```diff
@@ -38,36 +39,37 @@ import Foundation
 /// Its purpose is to allow processes to register with a cloud to be notified
 /// of updates to the channel, implementing a lightweight publish-subscribe
 /// protocol for RSS feeds.
-/// 
+///
 /// <cloud domain="rpc.sys.com" port="80" path="/RPC2" registerProcedure="myCloud.rssPleaseNotify" protocol="xml-rpc" />
-/// 
+///
 /// In this example, to request notification on the channel it appears in,
 /// you would send an XML-RPC message to rpc.sys.com on port 80, with a path
 /// of /RPC2. The procedure to call is myCloud.rssPleaseNotify.
-/// 
+///
 /// A full explanation of this element and the rssCloud interface is here:
 /// http://cyber.law.harvard.edu/rss/soapMeetsRss.html#rsscloudInterface
-public class Cloud {
-
-    /// The domain to register notification to.
-    public var domain: String?
-
-    /// The port to connect to.
-    public var port: Int?
-
-    /// The path to the RPC service. e.g. "/RPC2".
-    public var path: String?
-
-    /// The procedure to call. e.g. "myCloud.rssPleaseNotify" .
-    public var registerProcedure: String?
-
-    /// The `protocol` specification. Can be HTTP-POST, XML-RPC or SOAP 1.1 -
-    /// Note: "protocol" is a reserved keyword, so `protocolSpecification`
-    /// is used instead and refers to the `protocol` attribute of the `cloud`
-    /// element.
-    public var protocolSpecification: String?
-
-}
+//public class Cloud: NSManagedObject {
```

Comment:
lol why is this all commented out?

Reply:
i was scared to delete the original swift files lol